src/Compilers/CSharp/Test/Semantic/Semantics/StaticNullChecking_TypeInference.cs

```diff
@@ -1140,9 +1140,15 @@ static void G(string x, string? y)
                 references: new[] { ValueTupleRef, SystemRuntimeFacadeRef },
                 parseOptions: TestOptions.Regular8);
             comp.VerifyDiagnostics(
+                // (7,11): warning CS8620: Nullability of reference types in argument of type '(string x, string? y)' doesn't match target type '(string?, string?)' for parameter 't' in '(string?, string?) C.F<string?>((string?, string?) t)'.
```

Comment:
I didn't understand this warning. Passing a `(string, string?)` into a `(string?, string?)` seems safe.

Reply:
Good catch. This is a regression although we were previously not reporting a warning because the nullability of the tuple elements was ignored. For now, I've reverted the expected diagnostics for this test and the test below and marked the tests as skipped with a `// PROTOTYPE` comment.